packages/flutter/lib/src/widgets/form.dart

```diff
@@ -478,6 +493,15 @@ class FormField<T> extends StatefulWidget {
   /// not an error is displayed, either wrap the  [TextFormField] in a fixed
   /// height parent like [SizedBox], or set the [InputDecoration.helperText]
   /// parameter to a space.
+  ///
+  /// Note: Using [forceErrorText] will bypasses the regular
+  /// validation process performed by the [validator] function. This allows you to
+  /// handle error conditions or show specific error messages manually.
+  ///
+  /// See also:
+  ///
+  ///  * [forceErrorText], which may force the error state directly.
+  ///
```

Comment:
I think this "See also" is unnecessary since it restates the previous paragraph. I would do one or the other.

Reply:
I removed this too.